mailpoet/assets/js/src/newsletters/listings/standard.jsx

```diff
@@ -187,6 +187,25 @@ class NewsletterListStandardComponent extends Component {
     };
   }
 
+  componentDidMount() {
+    // Send event that the MP wizard was loaded via the WooCommerce home page to Mixpanel and delete the corresponding setting.
+    // We need to do this after the user completes the wizard, since before that tracking is not enabled.
+    if (window.mailpoet_track_wizard_loaded_via_woocommerce) {
+      MailPoet.trackEvent(
+        'User opened the MailPoet setup task in WooCommerce > Home',
+        {
+          'WooCommerce version': MailPoet.woocommerceVersion,
+        },
+      );
+      MailPoet.Ajax.post({
+        api_version: window.mailpoet_api_version,
+        endpoint: 'settings',
+        action: 'set',
```

Comment:
Ideally, I would like to remove the setting instead of changing its value. But it seems there is no method in our API to do that and I was not sure if I should add one. Thoughts?

I also considered using WP options, but it seems there is no WP REST API endpoint for options.

Reply:
I see that [the SettingsController has a method for deleting a setting](https://github.com/mailpoet/mailpoet/blob/926620e8f82ca691f1c11f6e973a2f86a6117522/mailpoet/lib/Settings/SettingsController.php#L117) so we could add the endpoint.

Reply:
I considered this option, but I wasn't sure if that was a desirable endpoint or not. Since you also think it is a good approach, I added it in a new commit.